R/AllGenerics.R

```diff
@@ -1,6 +1,7 @@
 setGeneric("bin", function(object, ...) standardGeneric("bin"))
 setGeneric("clean", function(object, ...) standardGeneric("clean"))
 setGeneric("compareSpectra", function(object1, object2, ...) standardGeneric("compareSpectra"))
+setGeneric("compareChromatograms", function(x, y, ...) standardGeneric("compareChromatograms"))
```

Comment:
Seeing these generics, I am wondering if `compareSpectra()`, `compareChromatograms()` and probably other shouldn't be moved to `ProtGenerics`. The latter is defined in `MSnbase` and `Spectra` already. 
```r
> MSnbase::compareSpectra
standardGeneric for "compareSpectra" defined from package "MSnbase"

function (object1, object2, ...) 
standardGeneric("compareSpectra")
<bytecode: 0x563351276228>
<environment: 0x563350fb3438>
Methods may be defined for arguments: object1, object2
Use  showMethods(compareSpectra)  for currently available ones.
> Spectra::compareSpectra
standardGeneric for "compareSpectra" defined from package "Spectra"

function (x, y, ...) 
standardGeneric("compareSpectra")
<bytecode: 0x563347e2f818>
<environment: 0x563347e21d20>
Methods may be defined for arguments: x, y
Use  showMethods(compareSpectra)  for currently available ones.
```

Reply:
Also, I would suggest to name the arguments as in `Spectra`.

Reply:
Very good point @lgatto . I will add `compareSpectra`, `compareChromatograms` and also `bin` to `ProtGenerics` - they are all implemented also in `Spectra`. Also, I agree on a common way to name parameters, but I would change from `object1, object2` to `x, y` as we have it in `Spectra`.

ProtGenerics PR will come.

Reply:
ProtGenerics PR: https://github.com/lgatto/ProtGenerics/pull/15